Dockerfile

```diff
@@ -9,4 +11,29 @@ RUN apt-get update \
         snmp \
     && docker-php-ext-install -j$(nproc) mysqli snmp \
     && docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ \
-    && docker-php-ext-install -j$(nproc) gd
\ No newline at end of file
+    && docker-php-ext-install -j$(nproc) gd \
+    && rm -rf /var/lib/apt/lists/* \
+    && apt-get clean
+
+RUN docker-php-ext-install zip
+
+ENV COMPOSER_ALLOW_SUPERUSER 1
+ENV COMPOSER_HOME /tmp
+ENV COMPOSER_VERSION 1.6.2
+
+RUN curl -s -f -L -o /tmp/installer.php https://raw.githubusercontent.com/composer/getcomposer.org/b107d959a5924af895807021fcef4ffec5a76aa9/web/installer \
+ && php -r " \
+    \$signature = '544e09ee996cdf60ece3804abc52599c22b1f40f4323403c44d44fdfdd586475ca9813a858088ffbc1f233e9b180f061'; \
```

Comment:
Wouldn't it make sense to put the signature into an Env var as well?
Does the signature relates to the specific Composer version or to the installer?
So is the signature changing, when installing a newer version of composer?

Reply:
I don't know. That's probably something to discuss at https://github.com/composer/docker/, which is where I took that part from. 😄